.github/workflows/robots-generator-deploy-dev.yml

```
@@ -0,0 +1,76 @@
+name: Robots Generator | Deploy | DEV
+
+on:
+  push:
+    branches: [ft/robots-generator]
+    paths:
+      - "apps/robots-generator/**"
+      - "Dockerfile.robots-generator"
+      - ".github/workflows/robots-generator-dev.yml"
+
+concurrency:
+  group: "${{ github.workflow }} @ ${{ github.ref }}"
+  cancel-in-progress: true
+
+env:
+  DOKKU_REMOTE_BRANCH: "master"
+  DOKKU_REMOTE_URL: "ssh://azureuser@ui-1.dev.codeforafrica.org"
+  IMAGE_NAME: "codeforafrica/robots-generator"
+  NEXT_PUBLIC_APP_URL: "https://robots-generator-ui.dev.codeforafrica.org"
+  GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+  APP_NAME: robots-generator-ui
+
+jobs:
+  deploy:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        node-version: [18]
```

```suggestion
        node-version: [20]
```

Lets also make sure we're using latest versions of actions (or at least exact version used in other apps).